worlds/sc2/item/item_groups.py

```diff
@@ -826,6 +907,46 @@ def get_all_group_names(cls) -> typing.Set[str]:
     + protoss_generic_upgrades
     + lotv_war_council_upgrades
 )
+item_name_groups[ItemGroupNames.PROTOSS_SC1_UNITS] = [
+    item_names.ZEALOT,
+    item_names.DRAGOON,
+    item_names.HIGH_TEMPLAR,
+    item_names.DARK_TEMPLAR,
+    item_names.DARK_ARCHON,
+    item_names.DARK_TEMPLAR_DARK_ARCHON_MELD,
+    # No shuttle
+    item_names.REAVER,
+    item_names.OBSERVER,
+    item_names.SCOUT,
+    item_names.CARRIER,
+    item_names.ARBITER,
+    item_names.CORSAIR,
+]
+item_name_groups[ItemGroupNames.PROTOSS_SC1_BUILDINGS] = [
+    item_names.PHOTON_CANNON,
+    item_names.SHIELD_BATTERY,
+]
+item_name_groups[ItemGroupNames.PROTOSS_LADDER_UNITS] = [
+    item_names.ZEALOT,
+    item_names.STALKER,
+    item_names.SENTRY,
+    item_names.ADEPT,
+    item_names.HIGH_TEMPLAR,
+    item_names.DARK_TEMPLAR,
+    item_names.DARK_TEMPLAR_ARCHON_MERGE,
+    # No shuttle
```

Comment:
SC2 ladder has no shuttles, thus this comment belongs only to SC1 variant (could've been copy-paste artifact)